src/test/java/uk/gov/justice/hmpps/casenotes/controllers/ResourceTest.kt

```diff
@@ -27,7 +28,7 @@ abstract class ResourceTest {
   @Autowired
   internal lateinit var jwtHelper: JwtAuthHelper
 
-  fun addBearerAuthorisation(user: String, roles: List<String>?): Consumer<HttpHeaders> {
+  fun addBearerAuthorisation(user: String, roles: List<String>? = listOf()): Consumer<HttpHeaders> {
```

Comment:
No need for nullable any more?

Reply:
agreed, will fix in next PR.  Wanted to change one of the resource tests anyway to Kotlin